bl/evaluation/evaluator_test.go

```diff
@@ -119,21 +142,20 @@ func TestEvaluate(t *testing.T) {
 
 			mockedCliClient.On("RequestEvaluation", mock.Anything).Return(tt.mock.cliClient.requestEvaluation.response, tt.mock.cliClient.requestEvaluation.err)
 
-			evaluator := &Evaluator{
-				cliClient: mockedCliClient,
-				osInfo:    tt.args.osInfo,
-			}
+			//evaluator := &Evaluator{
```

Comment:
Delete unused comments, we have git if we will want to recover it someday

Reply:
I fixed the tests - now it's not commented in